pkg/term/windows/ansi_test.go

```diff
@@ -386,3 +388,157 @@ func TestSplitChunkUnicode(t *testing.T) {
 		helpsTestOutputSplitThreeChunks(t, data)
 	}
 }
+
+func helpsTestParseInt16OrDefault(t *testing.T, expectedValue int16, shouldFail bool, input string, defaultValue int16, format string, args ...string) {
+	value, err := parseInt16OrDefault(input, defaultValue)
+	if nil != err && !shouldFail {
+		t.Errorf("Unexpected error returned %v", err)
+		t.Errorf(format, args)
+	}
+	if nil == err && shouldFail {
+		t.Errorf("Should have failed as expected\n\tReturned value = %d", value)
+		t.Errorf(format, args)
+	}
+	if expectedValue != value {
+		t.Errorf("The value returned does not macth expected\n\tExpected:%v\n\t:Actual%v", expectedValue, value)
+		t.Errorf(format, args)
+	}
+}
+
+func TestParseInt16OrDefault(t *testing.T) {
+	// empty string
+	helpsTestParseInt16OrDefault(t, 0, false, "", 0, "Empty string returns default")
+	helpsTestParseInt16OrDefault(t, 2, false, "", 2, "Empty string returns default")
+
+	// normal case
+	helpsTestParseInt16OrDefault(t, 0, false, "0", 0, "0 handled correctly")
+	helpsTestParseInt16OrDefault(t, 111, false, "111", 2, "Normal")
+	helpsTestParseInt16OrDefault(t, 111, false, "+111", 2, "+N")
+	helpsTestParseInt16OrDefault(t, -111, false, "-111", 2, "-N")
+	helpsTestParseInt16OrDefault(t, 0, false, "+0", 11, "+0")
+	helpsTestParseInt16OrDefault(t, 0, false, "-0", 12, "-0")
+
+	// ill formed strings
+	helpsTestParseInt16OrDefault(t, 0, true, "abc", 0, "Invalid string")
+	helpsTestParseInt16OrDefault(t, 42, true, "+= 23", 42, "Invalid string")
+	helpsTestParseInt16OrDefault(t, 42, true, "123.45", 42, "float like")
+
+}
+
+var allForeground = []int16{
```

Comment:
Tests now gone. The developer writing the new ANSI package is already working on proper tests with nice names. :)